joint_calling/upload_processor.py

```diff
@@ -0,0 +1,81 @@
+""" This function prepares gCVF's uploaded to GCS, based off sample status
+    logged in a database, for further QC and downstream analysis.
+    The upload processor will determine when samples should be added
+    to existing MatrixTables where appropriate and which MatrixTables
+    they should be combined with in this case. Following a successful
+    run all uploaded files will be moved to archival storage."""
+
+from google.cloud import storage
+from google.api_core.exceptions import NotFound
+
+
+def copy_files(
+    sample_files: list, source_bucket_name: str, target_bucket_name: str
+) -> bool:
+    """Given a list of file names, a source bucket and
+    a target bucket, this function will copy the identified files
+    to their new location. The function assumes that the file names have
+    been validated at an earlier stage. It will return true when the
+    files have been all successfully copied and false if at least 1 was not."""
+
+    # Connecting to buckets
+    try:
+        storage_client = storage.Client()
+        source_bucket = storage_client.get_bucket(source_bucket_name)
+        target_bucket = storage_client.get_bucket(target_bucket_name)
+    except NotFound as invalid_details:
+        raise Exception('Bucket does not exist for this user.') from invalid_details
+
+    # Copying files from source bucket to target bucket
```

Comment:
This comment is misleading -- you're not copying anything yet.

Reply:
Moved the comment so it's hopefully more clear :)